operations/reservations.md

```diff
@@ -499,6 +499,39 @@ Adds the specified reservations as a single group. If `GroupId` is specified, ad
             "CompanyId": null,
             "Notes": "Test reservation",
             "TimeUnitCost": null,
+            "TimeUnitPrices": [
+                {
+                    "Index": 0
+                    "Amount": {
+                        "Currency": "GBP",
+                        "NetValue": 2.08,
+                        "GrossValue": 2.5,
```

Comment:
@vitasamek do we support this? shouldn't it be either net or gross but not both? Check other occurrences if necessary.

Reply:
If the PR should be immediately merged, then there should be still an old Cost structure.
If it can be merged with the fiscal changes, I will do it on Monday.
For the new structure of amount, you mixed Amount parameters and Amount DTO.
For parameters you can send just net or just Gross and TaxValues -> TaxCodes